File: 7-kyu/fix-my-phone-numbers!/javascript/solution.js
const isItANum = str => {
  const onlyNums = str.split("").filter(elem => /\d/.test(elem)).join("");
  if (/^0\d{10}$/.test(onlyNums)) {
    return onlyNums;
  }
  return "Not a phone number";
};

File: 7-kyu/fix-my-phone-numbers!/python/solution.py
import re

def is_it_a_num(s: str) -> str:
    filtered: str = ''.join(ch for ch in s if ch.isdigit())
    if re.match(r'^0\d{10}$', filtered) is None:
        return 'Not a phone number'
    return filtered
    
